thriftpy2/contrib/aio/socket.py

```diff
@@ -106,58 +105,23 @@ def __init__(self, host=None, port=None, unix_socket=None,
             self.ssl_context = None
             self.server_hostname = None
 
-    def _init_sock(self):
-        if self.unix_socket:
-            _sock = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
-        else:
-            _sock = socket.socket(self.socket_family, socket.SOCK_STREAM)
-            _sock.setsockopt(socket.IPPROTO_TCP, socket.TCP_NODELAY, 1)
-
-        # socket options
-        linger = struct.pack('ii', 0, 0)
-        _sock.setsockopt(socket.SOL_SOCKET, socket.SO_LINGER, linger)
-        _sock.setsockopt(socket.SOL_SOCKET, socket.SO_KEEPALIVE, 1)
-
-        self.raw_sock = _sock
-
-    def set_handle(self, sock):
-        self.raw_sock = sock
-
-    def set_timeout(self, ms):
```

Comment:
I think we still can implement this method, just set the instance fields, which can be used in open and read methods.